.travis.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     - env: "MODE=e2e"
     - env: "MODE=aot"
     - env: "MODE=payload"
-    - env: "MODE=prerender"
+    - env: "MODE=server-rendering"
```

Comment:
I'd prefer to keep this a `prerender` since there's not actually a "server" involved